cypress/tests/ui/flowchart/flowchart.cy.js

```diff
@@ -6,13 +6,14 @@ describe('Flowchart DAG', () => {
 
   beforeEach(() => {
     cy.enablePrettyNames(); // Enable pretty names using the custom command
+    cy.wait(500);
```

Comment:
We are using a lot of `wait` is there not a way to check for some condition instead?

Reply:
i can't think of anyway, i think because both of the setting modals and the hint modal would take less than 500 ms to disappear. that's why I put it there and wait until the modal is completely gone before trying to click on another item

Reply:
and also i wonder why it doesnt fail anywhere else but only in this PR 🤔